### ISSUE TITLE
Captures media keys even when no usable tab for it: prevents native app from using them

### ISSUE THREAD

**Issue by tdd** [msfeldstein/chrome-media-keys#17]
Hey guys,

So this ext is **the bomb**, thank you so much.  Being able to control my Grooveshark instantly from anywhere (Pause/Play mostly) is awesome.

OTOH, I cringe at the flip side of this: apparently you guys capture these keys even when you can't use them. If I don't have any supported webapp opened in Chrome, for instance, they still get sucked by your ext apparently, because iTunes can't seem to use them anymore.

My using iTunes manually (play, pause, etc.) doesn't grab the keys back. I have to close Chrome entirely for this.

I'm on OSX Mavericks with Chrome 39 dev, if that helps.

Any ideas?

Best,

**Comment by madvas** [comment]
:+1:  Hopefully, chrome will resolve this soon

**Comment by naartjie** [comment]
Just checking, you guys have all cast your votes on the chromium bug right? :-)

**Comment by tdd** [comment]
Sure :-)

**Comment by naartjie** [comment]
@pbrooko @cyanogilvie we need your votes please ;-)

**Comment by msfeldstein** [comment]
Yeah star the bug, but dont add +1 comments that don't add more info they
might need.
